content/kubermatic/main/architecture/compatibility/supported-versions/_index.en.md

```diff
@@ -31,22 +31,20 @@ these migrations.
 In the following table you can find the supported Kubernetes versions for the
 current KKP version.
 
-| KKP version | 1.24\* | 1.23\* | 1.22 | 1.21 | 1.20\*\* | 1.19\*\*   | 1.18\*\*   |
-| ----------- | ------ |------- | ---- | ---- | -------- | ---------- | ---------- |
-| 2.21.x      | ✓      | ✓      | ✓    | ✓    | -        | -          | -          |
-| 2.20.x      | -      | -      | ✓    | ✓    | ✓        | -          | -          |
-| 2.19.x      | -      | -      | ✓    | ✓    | ✓        | -          | -          |
-| 2.18.x      | -      | -      | ✓    | ✓    | ✓        | ✓          | -          |
-| 2.17.x      | -      | -      | -    | ✓    | ✓        | ✓          | ✓          |
-| 2.16.x      | -      | -      | -    | -    | ✓        | ✓          | ✓          |
-
-\* Kubernetes 1.24 and 1.23 are currently not supported on ARM64 clusters with Canal CNI
-and kube-proxy running in the IPVS mode.
-
-\*\* Kubernetes 1.18, 1.19 and 1.20 releases have reached End-of-Life (EOL). We
-strongly recommend upgrading to a supported Kubernetes release as soon as
-possible.
-
-Upgrades from a previous Kubernetes version are generally supported whenever a
-version is marked as supported, for example KKP 2.17 supports updating clusters
-from Kubernetes 1.18 to 1.21.
+| KKP version  | 1.25     | 1.24[^1] | 1.23[^1] | 1.22[^2] | 1.21[^2] | 1.20[^2] | 1.19[^2]   |
+| ------------ | -------- | -------- | -------- | -------- | -------- | -------- | ---------- |
+| 2.22.x[^not-out-yet] | ✓        | ✓        | ✓        | -        | -        | -        | -          |
+| 2.21.x       | -        | ✓        | ✓        | ✓        | ✓        | -        | -          |
+| 2.20.x       | -        | -        | -        | ✓        | ✓        | ✓        | -          |
+| 2.19.x       | -        | -        | -        | ✓        | ✓        | ✓        | -          |
+| 2.18.x[^3]   | -        | -        | -        | ✓        | ✓        | ✓        | ✓          |
+
+[^1]: Kubernetes 1.24 and 1.23 are currently not supported on ARM64 clusters with Canal CNI and kube-proxy running in the IPVS mode.
+
+[^2]: Kubernetes 1.19, 1.20, 1.21 and 1.22 releases have reached End-of-Life (EOL). We strongly recommend upgrading to a supported Kubernetes release as soon as possible.
```

Comment:
Technically speaking, 1.22 is EOL but one more patch release is expected in December to include the registry change. I think we can declare it EOL in our docs but just a heads up to expect one more release.

Reply:
Yeah, I'd like to stay in line with official upstream communication, and https://kubernetes.io/releases/ says it was EOL'd 2022-10-28, so I think it makes sense to communicate it as EOL'd.

Reply:
Yes, I agree -- my comment was just a heads up that we're aware of this.